CHANGELOG.md

```diff
@@ -95,6 +71,7 @@ Keep it human-readable, your future self will thank you!
 - Renamed frequency keys in callbacks configuration. [#118](https://github.com/ecmwf/anemoi-training/pull/118)
 - Modified training configuration to support max_steps and tied lr iterations to max_steps by default [#67](https://github.com/ecmwf/anemoi-training/pull/67)
 - Merged node & edge trainable feature callbacks into one. [#135](https://github.com/ecmwf/anemoi-training/pull/135)
+- Increase the default MlFlow HTTP max retries [#111](https://github.com/ecmwf/anemoi-training/pull/111)
```

Comment:
Did you mean this to move?

Reply:
Yes, the Changelog had two "Changed" headings, so since I had to fix it anyways, I fixed this as well.